Source/WebKit/UIProcess/API/haiku/APIWebsiteDataStoreHaiku.cpp

```diff
@@ -28,12 +28,13 @@
 
 #include <wtf/FileSystem.h>
 
-#include "NotImplemented.h"
+#include <WebCore/NotImplemented.h>
 
 namespace API {
 
 String WebsiteDataStore::defaultApplicationCacheDirectory()
 {
+	fprintf(stderr,"error");
```

Comment:
Please avoid this kind of logs, it's hard to know where they come from, I spent time cleaning them up for you, don't reintroduce them.